.travis.yml

```diff
@@ -12,10 +12,12 @@ cache:
 
 install:
   - gem install rubocop
+  - gem install rake
```

Comment:
You are running `bundle exec rake` and rake isn't part of the Gemfile. Perhaps, add both rake and rubocop as dependencies of the Gem.